packages/invoices/src/InboundShipment/ListView/ListView.tsx

```diff
@@ -59,7 +57,7 @@ export const InboundListView: FC = () => {
       [
         'totalAfterTax',
         {
-          accessor: ({ rowData }) => c(rowData.pricing.totalAfterTax).format(),
+          accessor: ({ rowData }) => rowData.pricing.totalAfterTax,
```

Comment:
the column is expecting a number, and if you pass text it shows `''` 😁 
also, the currency cell is already using the intl currency formatter